pypesto/ensemble/constants.py

```diff
@@ -32,6 +32,9 @@
 STANDARD_DEVIATION = 'std'
 PERCENTILE = 'percentile'
 SUMMARY = 'summary'
+CHI2 = 'chi2'
+WEIGHTED_SIGMA = 'weighted_sigma'
+CONDITIONS = 'conditions'
```

Comment:
Is this used?

Reply:
only weighed sigma as it seems. deleted the others.